pkg/services/authz/rbac_settings.go

```diff
@@ -48,6 +51,8 @@ func readAuthzClientSettings(cfg *setting.Cfg) (*authzClientSettings, error) {
 
 	s.remoteAddress = authzSection.Key("remote_address").MustString("")
 	s.certFile = authzSection.Key("cert_file").MustString("")
+	s.cacheTTL = authzSection.Key("cache_ttl").MustDuration(30 * time.Second)
```

Comment:
can we set this from `shortCacheTTL`

Reply:
Discussed with you, the const is defined in another package.